Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gliderlabs/alpine
+
+COPY requirements.txt /opt/elodie/requirements.txt
+COPY docs/requirements.txt /opt/elodie/docs/requirements.txt
+COPY elodie/tests/requirements.txt /opt/elodie/elodie/tests/requirements.txt
+WORKDIR /opt/elodie
+
+RUN apk add --update boost-python ca-certificates exiftool exiv2 make python py-pip && \
```

Comment:
The main reason I did things this way is because there are (or were, at the time that I wrote this) dependencies that were required for compilation, but only for compilation. Splitting up the step of installing those dependencies would inflate the size of the resulting image, when they were only needed temporarily for building that dependency.

I could see splitting out the other dependencies, though, into a separate statement.